pages/dashboard/projects.vue

```diff
@@ -319,179 +319,146 @@ import SaveIcon from '~/assets/images/utils/save.svg'
+        return 0
+      })
+      break
+    case 'Type':
+      sortedArray = projects.slice().sort((a, b) => {
+        if (a.project_type < b.project_type) {
+          return -1
         }
-
-        if (!baseData.discord_url?.length ?? 1 > 0) {
-          delete baseData.discord_url
+        if (a.project_type > b.project_type) {
+          return 1
         }
+        return 0
+      })
+      break
+    default:
+      break
+  }
 
-        await useBaseFetch(
-          `projects?ids=${JSON.stringify(this.selectedProjects.map((x) => x.id))}`,
-          {
-            method: 'PATCH',
-            body: baseData,
-          }
-        )
-
-        this.$refs.editLinksModal.hide()
-        this.$notify({
-          group: 'main',
-          title: 'Success',
-          text: "Bulk edited selected project's links.",
-          type: 'success',
-        })
-        this.selectedProjects = []
-
-        this.editLinks.issues.val = ''
-        this.editLinks.source.val = ''
-        this.editLinks.wiki.val = ''
-        this.editLinks.discord.val = ''
-        this.editLinks.issues.clear = false
-        this.editLinks.source.clear = false
-        this.editLinks.wiki.clear = false
-        this.editLinks.discord.clear = false
-      } catch (e) {
-        this.$notify({
-          group: 'main',
-          title: 'An error occurred',
-          text: e,
-          type: 'error',
-        })
-      }
-    },
-  },
-})
+  if (descending) {
+    sortedArray = sortedArray.reverse()
+  }
+
+  return sortedArray
+}
+
+async function bulkEditLinks() {
+  try {
+    const baseData = {
+      issues_url: editLinks.issues.clear ? null : editLinks.issues.val.trim(),
+      source_url: editLinks.source.clear ? null : editLinks.source.val.trim(),
+      wiki_url: editLinks.wiki.clear ? null : editLinks.wiki.val.trim(),
+      discord_url: editLinks.discord.clear ? null : editLinks.discord.val.trim(),
+    }
+
+    if (!baseData.issues_url?.length ?? 1 > 0) {
+      delete baseData.issues_url
+    }
+
+    if (!baseData.source_url?.length ?? 1 > 0) {
+      delete baseData.source_url
+    }
+
+    if (!baseData.wiki_url?.length ?? 1 > 0) {
+      delete baseData.wiki_url
+    }
+
+    if (!baseData.discord_url?.length ?? 1 > 0) {
+      delete baseData.discord_url
+    }
+
+    await useBaseFetch(`projects?ids=${JSON.stringify(selectedProjects.value.map((x) => x.id))}`, {
+      method: 'PATCH',
+      body: baseData,
+    })
+
+    editLinksModal.value.hide()
+    data.$notify({
+      group: 'main',
+      title: 'Success',
+      text: "Bulk edited selected project's links.",
+      type: 'success',
+    })
+    selectedProjects.value = []
+
+    editLinks.value.issues.val = ''
+    editLinks.value.source.val = ''
+    editLinks.value.wiki.val = ''
+    editLinks.value.discord.val = ''
+    editLinks.value.issues.clear = false
+    editLinks.value.source.clear = false
+    editLinks.value.wiki.clear = false
+    editLinks.value.discord.clear = false
```

Comment:
There's also this block
```suggestion
    editLinks.issues.val = ''
    editLinks.source.val = ''
    editLinks.wiki.val = ''
    editLinks.discord.val = ''
    editLinks.issues.clear = false
    editLinks.source.clear = false
    editLinks.wiki.clear = false
    editLinks.discord.clear = false
```

Reply:
Fixed ^^